### PR TITLE
Add io.gitlab.floodlight.Presenter

### PR DESCRIPTION
**Pull request by flyingpimonster**
A GPL-licensed lyrics/presentation program designed for churches.

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build io.gitlab.floodlight.Presenter

File: io.gitlab.floodlight.Presenter.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/lib/debug",
+        "/share/gtk-doc",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
```
Comment (bilelmoussaoui) [inline io.gitlab.floodlight.Presenter.json]: Already in the runtime.

File: io.gitlab.floodlight.Presenter.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                "--disable-gtk-doc"
+            ],
+            "sources": [{
+                "type": "archive",
+                "url": "https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.16.2.tar.xz",
+                "sha256": "c724f612700c15a933c7356fbeabb0bb9571fb5538f8b1b54d4d2d94188deef2"
+            }]
+        },
+        {
+            "name": "libgee",
```
Comment (bilelmoussaoui) [inline io.gitlab.floodlight.Presenter.json]: Same

**Comment by bilelmoussaoui** [comment]
bot, build io.gitlab.floodlight.Presenter

File: io.gitlab.floodlight.Presenter.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        "/lib/debug",
+        "/share/gtk-doc",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "presenter",
+            "buildsystem": "meson",
+            "builddir": true,
```
Comment (bilelmoussaoui) [inline io.gitlab.floodlight.Presenter.json]:
```suggestion
```

**Comment by nedrichards** [comment]
Other than the minor change requested above it seems to work well and I look forward to being able to use it.

**Comment by bilelmoussaoui** [comment]
bot, build io.gitlab.floodlight.Presenter

**Pull request closed without merging**